lib/main.dart

```diff
@@ -51,9 +51,15 @@ void main(List<String> args) async {
       version =
           raw ? 'latest' : (await latestPublishedVersion(channel, flavor));
     } else if (sdk == 'main') {
-      channel = 'be';
-      version =
-          raw ? 'latest' : (await latestPublishedVersion(channel, flavor));
+      // TODO(b/299435467): Temporary forward compatibility that
```

Comment:
Can you replace this buganizer reference with a github issue ref?

Reply:
I could do that though the issue is just for myself to track the cleanups -- our team uses buganizer internally rather than github issues to manage our daily tasks and I never look at my github issues in practice. There's literally no content in the bug besides the title which is repeated here -- it just serves as an unique identifier that I can code search for to find all the relevant code locations that needs fixing. Do you feel I should file an issue in this repo about this cleanup instead?

Reply:
Gotcha. My main concern was that users w/o access to the bug would have some mystery about what it was tracking. I re-worded this comment slightly to clarify things (the buganizer issue just tracks the channel rename).